cmd/webhook-server/admission_controller.go

```diff
@@ -0,0 +1,154 @@
+/*
```

Comment:
Typically we use `%q` instead of `%s` to print variables in logs.
Use errors package instead of fmt.Error

Reply:
fixed